language-extensions/dotnet-core-CSharp/sample/regex/README.md

```diff
@@ -153,7 +153,7 @@ If you are using Windows, follow the steps below to create an external language
 
 1. Create a .zip file containing the extension.
 
-    Follow this [tutorial]((../../../../README.md)) to create **dotnet-core-CSharp-lang-extension.zip**. This file contains the **nativecsharpextension.dll** and **Microsoft.SqlServer.CSharpExtension.dll**.
+    Follow this [tutorial](../../../../README.md) to create **dotnet-core-CSharp-lang-extension.zip**. This file contains the **nativecsharpextension.dll** and **Microsoft.SqlServer.CSharpExtension.dll**.
```

Comment:
Good catch! 
Can you actually change the path to the point to the .NET Core language extension README.md instead of the root level README.
Basically, two parent level up
(../../README.md)

Reply:
the other PR https://github.com/microsoft/sql-server-language-extensions/pull/26/files fixes this.